src/ontology/mondo-ingest.Makefile

```diff
@@ -598,6 +598,22 @@ update-jinja-sparql-queries:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
+
+#################
+##### Ad hoc ####
+#################
+tmp/ORDO_en_4.4.owl:
+	wget https://www.orphadata.com/data/ontologies/ordo/last_version/ORDO_en_4.4.owl -O $@
+
+tmp/ordo-subsets.tsv: tmp/ORDO_en_4.4.owl
+	$(ROBOT) query -i $< --query ../sparql/select-ordo-subsets.sparql $@
+
+reports/ordo-subsets.robot.template.tsv: tmp/ordo-subsets.tsv tmp/mondo.sssom.tsv
```

Comment:
## Google sheet of template
See results: [google sheet](https://docs.google.com/spreadsheets/d/1UB0L62Rd2ZhqG20-3C5eNAtYeO9D-N2pe01e6LvRdRA/edit#gid=976194052&fvid=313621884)

Reply:
This looks amazing!